.gitignore

```diff
@@ -97,10 +97,12 @@ ENV/
 
 # Compiled Frontend
 donate/frontend/*
-/.vscode
 
 # PyCharm
 .idea/
 
 # Cypress visual regression testing
 cypress/videos
+
+# VSCode project config
+.vscode/settings.json
```

Comment:
It would be nicer if we could say "everything except for launch.json", e.g. https://stackoverflow.com/a/987162

Reply:
made that change 👍